### PR TITLE
ST-2467 Modify user and group for all environments

### PR DESCRIPTION
**Pull request by narumiruna**
- Rename jovyan to aurora
- Add group aurora with gid 2000 
- Add user aurora to group aurora
- Add group aurora to sudoers

### CODE REVIEW

File: base/base-notebook/Dockerfile [hunk 1]
```diff
@@ -35,7 +35,7 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 # Configure environment
 ENV CONDA_DIR=/opt/conda \
     SHELL=/bin/bash \
-    NB_USER=jovyan \
+    NB_USER=aurora \
     NB_UID=1000 \
     NB_GID=100 \
```
Comment (kkdai) [inline base/base-notebook/Dockerfile]: 原來的 gid 給 100, 這樣跟 @c9s 改得權限不同步.   aurora uid=1000, but gid=2000. 

PR 1254

Comment (narumiruna) [reply base/base-notebook/Dockerfile]: `groupadd -g 2000 aurora`
`useradd -m -s /bin/bash -N -u $NB_UID -g 2000 $NB_USER`
有 `-g 2000`, useradd 的同時有順便加入 2000

**Comment by kkdai** [comment]
Per discussion, it would be better first apply the related change on pytorch 0.4.1  
Make sure it works well on aurora dev and try to replace it on prod (w/o fix permission)
Then apply to rest of all.

File: env/pytorch/0.4.1/Dockerfile [hunk 2]
```diff
@@ -29,18 +29,18 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 
 # Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
```
Comment (c9s) [inline env/pytorch/0.4.1/Dockerfile]: 統一用 NB_USER, NB_UID, NB_GID ?

File: env/pytorch/0.4.1/Dockerfile.gpu [hunk 3]
```diff
@@ -27,19 +27,20 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
...
 
+# Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
+    GROUP_NAME=aurora \
+    USER_ID=1000 \
+    GROUP_ID=2000 \
```
Comment (c9s) [inline env/pytorch/0.4.1/Dockerfile.gpu]: 統一用 NB_USER, NB_UID, NB_GID ?

File: env/caffe/1.0/Dockerfile.gpu [hunk 4]
```diff
@@ -88,18 +88,18 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 
 # Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
+    GROUP_NAME=aurora \
+    USER_ID=1000 \
+    GROUP_ID=2000 \
```
Comment (c9s) [inline env/caffe/1.0/Dockerfile.gpu]: 統一用 NB_USER, NB_UID, NB_GID ?

File: env/caffe/1.0/Dockerfile [hunk 5]
```diff
@@ -83,18 +83,18 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 
 # Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
+    GROUP_NAME=aurora \
+    USER_ID=1000 \
+    GROUP_ID=2000 \
```
Comment (c9s) [inline env/caffe/1.0/Dockerfile]: 統一用 NB_USER, NB_UID, NB_GID ?

File: env/caffe-ssd/1.0/Dockerfile.gpu [hunk 6]
```diff
@@ -32,18 +32,18 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 
 # Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
+    GROUP_NAME=aurora \
+    USER_ID=1000 \
+    GROUP_ID=2000 \
```
Comment (c9s) [inline env/caffe-ssd/1.0/Dockerfile.gpu]: 統一用 NB_USER, NB_UID, NB_GID ?

File: env/caffe-ssd/1.0/Dockerfile [hunk 7]
```diff
@@ -32,18 +32,18 @@ RUN wget --quiet https://github.com/krallin/tini/releases/download/v0.10.0/tini
 
 # Configure environment
 ENV SHELL=/bin/bash \
-    NB_USER=aurora \
-    NB_UID=1000 \
-    NB_GID=100 \
+    USER_NAME=aurora \
+    GROUP_NAME=aurora \
+    USER_ID=1000 \
+    GROUP_ID=2000 \
```
Comment (c9s) [inline env/caffe-ssd/1.0/Dockerfile]: 統一用 NB_USER, NB_UID, NB_GID ?

**Pull request merged**